src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala

```diff
@@ -343,6 +343,69 @@ class DeltaSuite extends QueryTest
     }
   }
 
+  test("batch write: append, dynamic partition overwrite") {
+    withTempDir { tempDir =>
+      def data: DataFrame = spark.read.format("delta").load(tempDir.toString)
```

Comment:
nit: Remove the `DataFrame` type annotation.

Reply:
seems all other tests use this too? should keep style consistent?

Reply:
"be inconsistent if it's correct" :)

Reply:
i am tempted to fix these issues in a separate pullreq for all unit tests... the unnecessary type annotation, using spark.range, and removing unnecessary .toDF
might as well clean it all up

Reply:
removed it